include/netlink/xfrm/sa.h

```diff
@@ -141,8 +141,8 @@ extern int                      xfrmnl_sa_set_coaddr (struct xfrmnl_sa*, struct
 extern int                      xfrmnl_sa_get_mark (struct xfrmnl_sa*, unsigned int*, unsigned int*);
 extern int                      xfrmnl_sa_set_mark (struct xfrmnl_sa*, unsigned int, unsigned int);
 
-extern int                      xfrmnl_sa_get_sec_ctx (struct xfrmnl_sa*, unsigned int, unsigned int,
-                                                       unsigned int, unsigned int, char*);
+extern int                      xfrmnl_sa_get_sec_ctx (struct xfrmnl_sa*, unsigned int*, unsigned int*,
+                                                       unsigned int*, unsigned int*, char*);
```

Comment:
https://github.com/search?q=xfrmnl_sa_get_sec_ctx&type=Code
https://codesearch.debian.net/search?q=xfrmnl_sa_get_sec_ctx
https://searchcode.com/?q=xfrmnl_sa_get_sec_ctx
http://opensearch.krugle.org/document/search/#query=xfrmnl_sa_get_sec_ctx

Luckily all show no users of the function outside of the libnl source tree.